src/js/components/curriculum/CurriculumTable.js

```diff
@@ -20,9 +20,13 @@ export class CurriculumTable extends React.Component {
   }
 
   loadCurriculumContent() {
-    const { hasBeenFetched, curriculumList } = this.props.curriculum;
+    let { hasBeenFetched, curriculumList } = this.props.curriculum;
```

Comment:
Declaring props as `const` during destructuring serves to ensure the values are never mistakenly mutated. Is there a reason you switched to `let` ?